util/helm.go

```diff
@@ -122,3 +122,8 @@ func CopyDir(src, dst string) error {
 	filepath.Walk(src, walker)
 	return failure
 }
+
+// ExpandChartPath - Expand chart path from name and home directory
+func ExpandChartPath(dir, chartName string) string {
```

Comment:
I rabbit-holed on the directory structure functions.  Just making it a better name for now.  I'll work on the directory refactor next.